tests/integration-test/server/config.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const { join } = require('path');
+
+const PORT = 4000;
+
+module.exports = {
+  STORAGE_BASE_DIR: join(__dirname, './storage'),
+  API_TOKEN: 'my-very-special-key12345',
+  API_BASE_URL: `http://localhost:${PORT}/v1`,
+  PORT,
```

Comment:
feature request: add `TEST_REPORT_PATH` here and write the test report to this location